src/settings-ui/Settings.UI/Views/ColorPickerPage.xaml

```diff
@@ -17,133 +17,133 @@
         <StackPanel Orientation="Vertical"
```

Comment:
@jay-o-way This page crashes with a XAML related error. For you as well? Due to the many formatting changes it's hard to check what was changed.